Problem: data-stream-as-disjoint-intervals
Submission ID: 1695695304
Submission Time: 2025-07-12 15:08:55
============================================================

State: SUCCESS
Status: Runtime Error
Language: Python

Runtime: N/A ms
Memory: 12.51 MB

Test Cases: 0/7 passed

Last Test Case:
Input: ["SummaryRanges","addNum","getIntervals","addNum","getIntervals","addNum","getIntervals","addNum","getIntervals","addNum","getIntervals"]
[[],[1],[],[3],[],[7],[],[2],[],[6],[]]
Expected: [null,null,[[1,1]],null,[[1,1],[3,3]],null,[[1,1],[3,3],[7,7]],null,[[1,3],[7,7]],null,[[1,3],[6,7]]]

============================================================
RAW SUBMISSION DATA
============================================================
{
  "status_code": 15,
  "lang": "python",
  "run_success": false,
  "runtime_error": "Line 82: TypeError: [<__main__.Interval object at 0x7f86ea53c1d0>] is not valid value for the expected return type integer[][]",
  "full_runtime_error": "TypeError: [<__main__.Interval object at 0x7f86ea53c1d0>] is not valid value for the expected return type integer[][]\n    raise TypeError(str(result) + \" is not valid value for the expected return type integer[][]\")\nLine 82 in __helper_select_method__ (Solution.py)\n    ctime, ans = __DriverSolution__().__helper_select_method__(method, params[index], obj)\nLine 120 in _driver (Solution.py)\n    _driver()\nLine 132 in <module> (Solution.py)",
  "status_runtime": "N/A",
  "memory": 12512000,
  "question_id": "352",
  "elapsed_time": 30,
  "compare_result": "0000000",
  "code_output": "",
  "std_output": "",
  "last_testcase": "[\"SummaryRanges\",\"addNum\",\"getIntervals\",\"addNum\",\"getIntervals\",\"addNum\",\"getIntervals\",\"addNum\",\"getIntervals\",\"addNum\",\"getIntervals\"]\n[[],[1],[],[3],[],[7],[],[2],[],[6],[]]",
  "expected_output": "[null,null,[[1,1]],null,[[1,1],[3,3]],null,[[1,1],[3,3],[7,7]],null,[[1,3],[7,7]],null,[[1,3],[6,7]]]",
  "task_finish_time": 1752358135181,
  "task_name": "judger.judgetask.Judge",
  "finished": true,
  "total_correct": 0,
  "total_testcases": 7,
  "runtime_percentile": null,
  "status_memory": "N/A",
  "memory_percentile": null,
  "pretty_lang": "Python",
  "submission_id": "1695695304",
  "status_msg": "Runtime Error",
  "state": "SUCCESS"
}